common.yaml.jinja

```
@@ -18,6 +18,7 @@ services:
       PGUSER: &dbuser "{{ postgres_username }}"
       DB_FILTER: "{{ odoo_dbfilter }}"
       PROXY_MODE: "{% if odoo_proxy %}true{% else %}false{% endif %}"
+      LIST_DB: "{{ odoo_listdb | to_json }}"
```

Sorry, I'm already thinking on copier 6.

```suggestion
      LIST_DB: "{{ odoo_listdb | tojson }}"
```

---

No problem :wink: Already changed :heavy_check_mark: